runbot_restore_db/runbot.py

```diff
@@ -64,7 +64,10 @@ def job_25_restore(self, cr, uid, build, lock_path, log_path):
     def job_26_upgrade(self, cr, uid, build, lock_path, log_path):
         if not build.repo_id.db_name:
             return 0
-        to_test = build.modules if build.modules else 'all'
+        if build.repo_id.force_update_all:
```

Comment:
to_test = build.modules if build.modules and not build.repo_id.force_update_all else 'all'